src/main/java/com/jcabi/maven/plugin/AjcMojo.java

```diff
@@ -106,6 +106,17 @@
     private transient File classesDirectory;
 
     /**
+     * Directory in which uwoven classes are copied.
+     * @checkstyle MemberNameCheck (10 lines)
+     */
+    @Parameter(
+        required = false,
+        readonly = false,
+        defaultValue = "${project.build.outputDirectory}"
```

Comment:
@mkordas I don't think so. Think of the relation between the two directories: we should copy classes from ``classesDirectory`` to ``unwovenClassesDir`` only if the 2 differ. If none of them is specified, then they must have the same default value, so nothing is copied. 

It's actually a matter of "triggering" the copy by specifying the ``unwovenClassesDir`` different than ``classesDirectory`` . If we give a default value to ``unwovenClassesDir`` that's different from ``classesDirectory`` then it will be like a blackbox and the users of the plugin (others than us) might wonder why the ``target/unwoven`` folder and copy of ``.class`` files. 

The other case, when ``classesDirectory`` is specified and ``unwovenClassesDir`` is not, will result in ``classesDirectory`` containing woven classes and ``target/classes`` containing unwoven classes. 

The default hardcoded ``target/unwoven`` folder was only a suggestion of the author, but I think the current logic is the best.

I hope this makes sense to you as well. If not, feel free to ask more questions.